amgut/templates/sitebase.html

```diff
@@ -1,8 +1,8 @@
 {% from future.utils import viewitems %}
 {% from amgut import media_locale %}
 {% from amgut.connections import ag_data, redis %}
-{% set human_participants, animal_participants, environmental_samples, kit_verified = ag_data.get_menu_items(skid) %}
-{% set base_user_id = ag_data.get_user_for_kit(skid) %}
+{% set human_participants, animal_participants, environmental_samples, kit_verified = ag_data.get_menu_items(current_user) %}
+{% set base_user_id = ag_data.get_user_for_kit(current_user) %}
```

Comment:
This change is to use the current_user variable given by tornado itself instead of having to pass in skid to all templates. Makes calling render easier, as you don't have to pass in the skid any more. No changes to the rendered templates.